SimpleLottery/Model/Lottery.swift

```diff
@@ -8,27 +8,34 @@
 
 import Foundation
 
+
 struct Lottery {
     
-    static let price = 1000
-    static let numbersRange = 1...45
-    static let numbersMaximumCount = 6
-    
-    let numbers: [Int]
+    // MARK - init
     
     init(numbers: [Int] = []) {
         self.numbers = Lottery.generatedNumbers(with: numbers)
     }
     
+    // MARK: - internal
+    
+    static let price = 1000
```

Comment:
```suggestion
    enum Constant {
        static let price = 1000
        static let numberRange = 1...45
        static let numberMaximumCount = 6
    }
```

외부에서 상수로 쓰기위해 static으로 선언한거라면 이렇게 enum에 넣어서 묶어서 쓰이는게 좋지 않을까요?

Reply:
말씀해주신 것처럼 외부에서 상수로 쓰기 위함이 맞는데,
Lottery라는 구조체에서 또 enum으로 묶게 된다면 외부에서 쓸 때 문장이 길어질 것 같아 enum을 사용하지 않았습니다.

제 생각에는 Lottery의 가격, Lottery number의 범위, Lottery number를 가질 수 있는 최대 개수를 나타내는 3개의 변수가 Lottery라는 키워드에서 또 enum으로 묶일 필요성이 있나 싶습니다.

혹시 이런 상황에서 enum을 쓰면 얻게 되는 장점이 있을까요?

Reply:
저는 단어가 무분별하게만 아니면, 적당히 길어지는 것은 명확성보다는 중요하지 않다고 생각합니다!
지금은 3개일 지라도 서비스를 만들게 되면 3개이상으로 추가될 수 도 있으므로, enum으로 묶어서 쓰게 되면 static 변수 추가할때 관리도 편하고 가독성도 좋아지고 타인이 쓸 때도 좀 더 직관적으로 사용할 수있을 것같아요 (회사에서는 지금 저렇게 쓰고있어서 추천해봤어용 ㅎㅎ) . 그러나 스타일 차이니 불필요하다 싶으면 skip하도록 할게용

Reply:
좋은 의견 감사합니다.

Clean Code 37페이지 `불필요한 맥락을 없애라`에 아래와 같이 기술되어 있는데요.
> 일반적으로는 짧은 이름이 긴 이름보다 좋다. 단 의미가 분명한 경우에 한해서다. 이름에 불필요한 맥락을 추가하지 않도록 주의한다.

그래서 저는 저 부분을 enum으로 묶는 것에 대해서 불필요한 맥락이라고 생각했던 것 같아요.
그런데 현아님 말씀대로 상수로 써야할 조건들이 많아지면 enum으로 묶는 것이 훨씬 더 직관적으로 사용할 수 있을 것 같습니다.

여기에선 저렇게 두고,
더 확장해야 할 경우에는 현아님 조언을 꼭 기억하도록 하겠습니다.

Reply:
![image](https://user-images.githubusercontent.com/28696722/56580650-01f63a00-660e-11e9-9bb7-b31189d1c550.png)

전수열님의 코드스타일가이드에서 발췌해온 내용이에요 ㅎㅎ 
저보다 더 설명을 잘해주신것 같아서 혹 도움이 될까 추가적으로 올려드립니다.

Reply:
좋은 정보네요.
저 내용 그대로 현아님이 잘 설명해주셨어요!
저 가이드라인은 수열님 GitHub에 가면 볼 수 있나요?

Reply:
저희 팀에도 네이밍컨벤션이나 코드스타일 등을 정하려고해서 이거저거 찾아보고있는데요,
스타일쉐어 코드컨벤션이 굉장히 잘 정리되어있는거 같아서 좋네용 ㅎㅎ
https://github.com/StyleShare/swift-style-guide

요기 보시면 전문 보실수있어용!

Reply:
감사합니다~